esp-hal/src/dma/mod.rs

```diff
@@ -208,10 +208,31 @@ bitfield::bitfield! {
     pub struct DmaDescriptorFlags(u32);
```

Comment:
This probably shouldn't remain `#[doc(hidden)]`

Reply:
Ahh I knew I was missing something